wallet/src/de/schildbach/wallet/ui/main/WalletActivity.java

```diff
@@ -496,8 +213,6 @@ protected Dialog onCreateDialog(final int id, final Bundle args) {
             return createBackupWalletPermissionDialog();
         else if (id == DIALOG_RESTORE_WALLET_PERMISSION)
             return createRestoreWalletPermissionDialog();
-        //else if (id == DIALOG_RESTORE_WALLET)
-        //    return createRestoreWalletDialog();
         else if (id == DIALOG_TIMESKEW_ALERT)
             return createTimeskewAlertDialog(args.getLong("diff_minutes"));
         else if (id == DIALOG_VERSION_ALERT)
```

Comment:
These dialogs also seem to be not used, except for the low storage dialog. Do we need these:
```
private static final int DIALOG_BACKUP_WALLET_PERMISSION = 0;
private static final int DIALOG_RESTORE_WALLET_PERMISSION = 1;
private static final int DIALOG_TIMESKEW_ALERT = 3;
private static final int DIALOG_VERSION_ALERT = 4;
```
Perhaps we should get rid of them in the future and move the low storage alert to an `AdaptiveDialog`

Reply:
We need the timeskew alert because the system clock must be set to be similar to the Dash Network time otherwise the blockchain won't download.

Version Alert was never used and the permissions became obsolete with version 7.